### ISSUE TITLE
Root application adding spurious backslahes

### ISSUE THREAD

**Issue by drmarkwslater** [ganga-devs/ganga#894]
If you do a simple Root job as suggested in the help:
```
j = Job()
j.application = Root()
j.application.script = '~/analysis.C'
j.application.args = ['Minbias', 10]
j.submit()
```
It doesn't work because a backslash is appended to the script name:
```
Warning in <TApplication::GetOptions>: macro analysis.C\ not found
```

This can be seen in the python wrapper script:
```
appscriptpath = ['root.exe', '-b', '-q', 'analysis.C\\(\'"Minbias"\',10\\)']
```
I'm not sure where these are being added. it looks like its a strange python thing when printing lists:
```
>>> a = "\(''\)"
>>> b = [a]
>>> print a, b
\(''\) ["\\(''\\)"]
```

Maybe related to #37

**Comment by mesmith75** [comment]
This is only an issue for the interactive backend - it works fine for localhost. I think this is due to how we pass the arguments to the os instead of popen.

**Comment by mesmith75** [comment]
Fixed by #1340
